ci-operator/config/openshift/knative-client/openshift-knative-client-release-v0.26.0.yaml

```diff
@@ -0,0 +1,123 @@
+base_images:
+  base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+  ocp_ubi-minimal_8:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+binary_build_commands: |
+  TAG=v0.26.0 make install
+  TAG=v0.26.0 make build-cross
+build_root:
+  project_image:
+    dockerfile_path: openshift/ci-operator/build-image/Dockerfile
+canonical_go_repository: github.com/knative/client
+images:
+- dockerfile_path: openshift/ci-operator/knative-images/client/Dockerfile
+  from: base
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/kn
+  to: knative-client
+- dockerfile_path: openshift/ci-operator/knative-images/client/Dockerfile.cliartifacts
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/src/github.com/knative/client/kn-linux-amd64
+      - destination_dir: .
+        source_path: /go/src/github.com/knative/client/kn-linux-ppc64le
+      - destination_dir: .
+        source_path: /go/src/github.com/knative/client/kn-linux-s390x
+      - destination_dir: .
+        source_path: /go/src/github.com/knative/client/kn-linux-arm64
+      - destination_dir: .
+        source_path: /go/src/github.com/knative/client/kn-darwin-amd64
+      - destination_dir: .
+        source_path: /go/src/github.com/knative/client/kn-windows-amd64.exe
+      - destination_dir: .
+        source_path: /go/src/github.com/knative/client/LICENSE
+      - destination_dir: .
+        source_path: /go/src/github.com/knative/client/package_cliartifacts.sh
+    ocp_ubi-minimal_8:
+      as:
+      - registry.ci.openshift.org/ocp/ubi-minimal:8
+  to: kn-cli-artifacts
+- dockerfile_path: openshift/ci-operator/knative-test-images/helloworld/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/helloworld
+  to: knative-client-test-helloworld
+- dockerfile_path: openshift/ci-operator/knative-test-images/grpc-ping/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/grpc-ping
+  to: knative-client-test-grpc-ping
+- dockerfile_path: openshift/ci-operator/knative-test-images/servingcontainer/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/servingcontainer
+  to: knative-client-test-servingcontainer
+- dockerfile_path: openshift/ci-operator/knative-test-images/sidecarcontainer/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/sidecarcontainer
+  to: knative-client-test-sidecarcontainer
+promotion:
+  name: knative-v0.26.0
+  namespace: openshift
```

Comment:
@mgencur @markusthoemmes 

I've removed the whole chunk of `releases` configuratiion from here, e.g. the [previous release](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/knative-client/openshift-knative-client-release-v0.25.1.yaml#L85-L94). By the [CI docs](https://docs.ci.openshift.org/docs/architecture/ci-operator/#should-i-use-an-ephemeral-or-published-release) it seems to like an ignored branch, since we define the version in `cluster_claim`. Do you see any issues with that? In addition the whole block seems to be kind of wrong, e.g. `...stable ...4.8`  should be definitely sufficient.

Reply:
Yeah. I think that part is redundant when we use cluster_claim. No issues from my side.